src/components/NcActionButtonGroup/NcActionButtonGroup.vue

```diff
@@ -29,19 +29,22 @@ This should be used sparingly for accessibility.
 		<NcActions>
 			<NcActionButtonGroup name="Text alignment">
 				<NcActionButton aria-label="Align left"
-					@click="showMessage('Align left')">
+					:pressed="alignment === 'l'"
+					@click="alignment = 'l'">
 					<template #icon>
 						<AlignLeft :size="20" />
 					</template>
 				</NcActionButton>
 				<NcActionButton aria-label="Align center"
-					@click="showMessage('Align center')">
+					:pressed="alignment === 'c'"
+					@click="alignment = 'c'">
 					<template #icon>
 						<AlignCenter :size="20" />
 					</template>
 				</NcActionButton>
 				<NcActionButton aria-label="Align right"
-					@click="showMessage('Align Right')">
+					:pressed="alignment === 'r'"
+					@click="alignment = 'r'">
```

Comment:
This example represents a radio group (only one of the buttons can be selected). But with `aria-pressed` there is no semantic of a radio group. With a screen reader, a user can hear whether "Aligned center" is pressed or not pressed, but not that there is a group of 3 possible alignments

According to the [specification](https://w3c.github.io/aria/#aria-pressed), it is supposed to be used on a toggle button

Reply:
Maybe we need role=group on the button group component?

Reply:
`role=group` should be added (doing it in another PR), but it doesn't add radio semantics to the group, "only one of options is selected".

Reply:
But this is usage dependent if you want radio semantics or not, for this PR it is supporting toggle-buttons.
Having button groups as radio buttons is another topic (well part of the example here...). Because that would need changes on the button group component.

Reply:
So for the moment I would like to got with this one (for `NcActionButton`) and do a follow up for `NcActionButtonGroup`. As this PR does not change anything (related to a11y) for the later one.

Reply:
This PR adds a new prop for a new feature, which cannot be undone or changed much without a breaking change. That's why I wouldn't rush here without the need.